Tests/AppTests/Controllers/User/UserPhotoTest.swift

```diff
@@ -0,0 +1,206 @@
+import XCTest
+import Testing
+import HTTP
+import Core
+import Foundation
+import Multipart
+import FormData
+import Crypto
+
+@testable import Vapor
+@testable import App
+
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable force_try
+class UserPhotoTest: TestCase {
+
+  let photoPath = "user_photos/1/"
+  let testResouces = "Tests/Resources/"
+  var storedDir = ""
+  let fileManager = Foundation.FileManager()
+  var clientToken = ""
+  var headers: [HeaderKey: String] = [:]
+  var domain = ""
+
+  override func setUp() {
+    super.setUp()
+    do {
+      try drop.truncateTables()
+      storedDir = drop.config.publicDir + photoPath
+      clientToken = drop.config["app", "client-token"]?.string ?? ""
+      domain = drop.config["app", "domain"]!.string!
+      headers = [
+        "client-token": clientToken,
+        HeaderKey.contentType: TestConstants.Header.Value.multipartFormData
+      ]
+    } catch {
+      XCTFail("Droplet set raise exception: \(error.localizedDescription)")
+      return
+    }
+  }
+
+  func testThatUserPhotoIsUpdatedByFormDataSend() throws {
```

Comment:
may be `...isUpdatedUsingFormDataContent`

Reply:
ok, fixed